docs/generators/dart.md

```diff
@@ -19,7 +19,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |pubLibrary|Library name in generated code| |null|
 |pubName|Name in generated pubspec| |null|
 |pubVersion|Version in generated pubspec| |null|
-|serializationLibrary|Specify serialization library|<dl><dt>**native_serialization**</dt><dd>Use native serializer, backwards compatible</dd><dt>**json_serializable**</dt><dd>Use json_serializable</dd></dl>|native_serialization|
+|serializationLibrary|Specify serialization library|<dl><dt>**native_serialization**</dt><dd>Use native serializer, backwards compatible</dd><dt>**json_serializable**</dt><dd>Use json_serializable. Experimental and subject to breaking changes in minor releases</dd></dl>|native_serialization|
```

Comment:
> Experimental and subject to breaking changes in minor releases

What about `Experimental and subject to breaking changes without further notice` as you probably want to do breaking changes in patch releases as well (e.g. 5.1.0 => 5.1.1) ?

Reply:
done